doc/source/whatsnew/v0.22.0.rst

```diff
@@ -54,7 +54,7 @@ The default sum for empty or all-*NA* ``Series`` is now ``0``.
 
 *pandas 0.22.0*
 
-.. ipython:: python
+.. code-block:: ipython
 
```

Comment:
see above comments (alternatively okwarning is fine)

Reply:
okwarning really seems to be the easier way. Thanks